storage/kafkasql/src/main/java/io/apicurio/registry/storage/impl/kafkasql/values/ActionType.java

```diff
@@ -32,11 +32,7 @@
     DELETE(3),
     CLEAR(4),
     IMPORT(5),
-    RESET(6),
-    /**
-     * Deletes ALL user (tenant) data. Does not delete global data, such as log configuration.
-     */
-    DELETE_ALL_USER_DATA(7);
+    RESET(6);
```

Comment:
What do you think about this?  I wasn't sure we needed the `DELETE_ALL_USER_DATA` enum when we already have `DELETE` which, when combined with the `GlobalAction` type seemed sufficient.

Thoughts?

Reply:
No strong opinion, I think the original name is more explicit about what it's doing. Sometimes words can get too overloaded.

Reply:
Yeah I think I'll revert this change.